tools/hangar/__snapshots__/invalid.ts.snap

```diff
@@ -138,14 +138,25 @@ exports[`capture_reassignable.w > stderr 1`] = `
 `;
 
 exports[`cloud_function_expects_inflight.w > stderr 1`] = `
-"error: Expected type to be \\"inflight (any): any\\", but got \\"preflight (str): str\\" instead
+"error: Expected type to be \\"IFunctionHandler\\", but got \\"preflight (str): str\\" instead
```

Comment:
This is a regression in error messages - not sure the best way to handle this yet

Reply:
OK, figured out a way to get a reasonable-ish experience -- if there is an inflight method named "handle" on the interface, it will be displayed in parentheses